kolibri/plugins/learn/assets/src/views/ChannelCardGroupGrid.vue

```diff
@@ -56,6 +58,17 @@
         default: false,
       },
     },
+    computed: {
+      layoutSpan() {
+        let span = 3;
+        if ([0, 1, 2, 6].includes(this.windowBreakpoint)) {
+          span = 4;
+        } else if ([3, 4, 5].includes(this.windowBreakpoint)) {
+          span = 6;
+        }
```

Comment:
The use of numbers here is kind of ambiguous. I had to dive into the `KResponsiveWindow` code to get a grasp on what the numbers refer to and I'm not really sure exactly what these conditions translate to IRL after looking at that code.

It'd be really helpful here if you could add some commentary here explaining what this should translate to on the screen. It strikes me as odd that 6 is lumped in with 0-2 and anything bigger than 6 is lumped in as the smallest possible span.

Reply:
Also noting there are some other areas with this same logic.

Maybe it would be worth considering moving this logic into a shared parent component to `provide` it and you can `inject` it wherever it is used - rather than adding the same comments across a bunch of files but that'd probably be just fine too I think

Reply:
Hi @nucleogenesis, I have made the changes as recommended. Considering the odd '6', this is intentional and hopefully the comments added to the `layoutSpan` property make it clearer.